wpilibc/src/main/native/cpp/shuffleboard/ShuffleboardContainer.cpp

```diff
@@ -0,0 +1,167 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/shuffleboard/ShuffleboardContainer.h"
+
+#include <wpi/raw_ostream.h>
+
+#include "frc/shuffleboard/ComplexWidget.h"
+#include "frc/shuffleboard/ShuffleboardComponent.h"
+#include "frc/shuffleboard/ShuffleboardLayout.h"
+#include "frc/shuffleboard/SimpleWidget.h"
+
+using namespace frc;
+
+ShuffleboardContainer::ShuffleboardContainer(wpi::StringRef title)
+    : ShuffleboardValue(title) {}
+
+ShuffleboardContainer::ShuffleboardContainer(ShuffleboardContainer&& rhs)
```

Comment:
This doesn't look different than the implicit/defaulted move constructor, which does a member-wise move of all non-static data members?  If it's the same, let's use the implicit one.